cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -267,8 +267,8 @@ class EvaluatorCommand : CliktCommand(name = "evaluate", help = "Evaluate ORT re
             ortResultInput = ortResultInput.replaceConfig(config)
         }
 
-        val curations = FilePackageCurationProvider.from(packageCurationsFile, packageCurationsDir).packageCurations
-        if (curations.isNotEmpty()) {
+        if (packageCurationsFile != null || packageCurationsDir != null) {
```

Comment:
I'd propose to use `packageConfigurationOption != null` instead as it's shorter and nicely emphasizes that you want to do something as soon as the option is specified.